src/components/SelectList/SelectList.tsx

```diff
@@ -309,21 +311,54 @@ const MultiValueRemove = props => (
     </ReactSelectComponents.MultiValueRemove>
 );
 
+// eslint-disable-next-line react/prop-types
```

Comment:
Doesn't `react-select` export props for the `Menu` component? If they exist we could get rid of both the eslint and the ts comment

Reply:
good point 👍